### PR TITLE
docs: add sync workflow

### PR DESCRIPTION
**Pull request by bastiandoetsch**
### Description

_Provide description of this PR and changes, if linked Jira ticket doesn't cover it in full._

### Checklist

- [ ] Tests added and all succeed
- [ ] Linted
- [ ] CHANGELOG.md updated
- [ ] README.md updated, if user-facing

### Screenshots / GIFs

_Visuals that may help the reviewer. Please add screenshots for any UI change. GIFs are most welcome!_

### CODE REVIEW

File: .github/workflows/readme-sync.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+              gh pr create --title="$MESSAGE" --body="Automatic PR controlled by GitHub Action." --head $DESTINATION_BRANCH
+            fi
+            echo "PR exists, pushed changes to it."
+          else
+            echo "No documentation changes detected, exiting."
+          fi
+        env:
+          SOURCE_PATH: ./docs/docs/scm-ide-and-ci-cd-workflow-and-integrations/snyk-ide-plugins-and-extensions/jetbrains-plugins/README.md
+          FILE_TO_COMMIT: README.md
+          DESTINATION_REPOSITORY: snyk-eclipse-plugin
```
Comment (teodora-sandu) [inline .github/workflows/readme-sync.yml:57]: Should this be `snyk-intellij-plugin`?

File: .github/workflows/readme-sync.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            if [[ ! $(gh pr list --search "$MESSAGE" 2>&1 | grep -e "$MESSAGE";) ]]; then
+              echo "Creating PR"
+              gh pr create --title="$MESSAGE" --body="Automatic PR controlled by GitHub Action." --head $DESTINATION_BRANCH
+            fi
+            echo "PR exists, pushed changes to it."
+          else
+            echo "No documentation changes detected, exiting."
+          fi
+        env:
+          SOURCE_PATH: ./docs/docs/scm-ide-and-ci-cd-workflow-and-integrations/snyk-ide-plugins-and-extensions/jetbrains-plugins/README.md
```
Comment (teodora-sandu) [inline .github/workflows/readme-sync.yml:55]: Is this means to have double `docs`?

File: .github/workflows/readme-sync.yml (hunk 1 above)
Comment (bastiandoetsch) [reply .github/workflows/readme-sync.yml:57]: oh yeah

File: .github/workflows/readme-sync.yml (hunk 2 above)
Comment (bastiandoetsch) [reply .github/workflows/readme-sync.yml:55]: yes

File: .github/workflows/readme-sync.yml (hunk 1 above)
Comment (bastiandoetsch) [reply .github/workflows/readme-sync.yml:57]: yeah, pushed a fix

**Review by teodora-sandu — APPROVED** [review]

**Pull request merged**